system-int-test/test/cmr/system_int_test/virtual_product/virtual_product_test.clj

```diff
@@ -369,3 +373,21 @@
             (search/find-refs :granule {:entry-title (:entry-title vp-coll)
                                         :page-size 1000})))))))
 
+(deftest CMR-5848-umm-json-granule-virtual-product-ingest-test
+  (testing "ingest with Virtual-Product-Service with a umm-json granule"
+    (let [psa1 (data-umm-cmn/additional-attribute {:Name "source_granule_ur" :DataType "STRING" :Description "Test"})
+          collection (d/ingest-umm-spec-collection "GES_DISC" (data-umm-c/collection
+                                                                {:EntryTitle "OMI/Aura Surface UVB Irradiance and Erythemal Dose Daily L3 Global 1.0x1.0 deg Grid V003 (OMUVBd) at GES DISC"
+                                                                 :ShortName "OMUVBd_ErythemalUV"
+                                                                 :AdditionalAttributes [psa1]
+                                                                 :Version "003"}))
+          granule (dg/granule-with-umm-spec-collection
+                   collection
+                   (:concept-id collection)
+                   {:provider-id "GES_DISC"
+                    :collection-ref (umm-g/map->CollectionRef {:entry-title "OMI/Aura Surface UVB Irradiance and Erythemal Dose Daily L3 Global 1.0x1.0 deg Grid V003 (OMUVBd) at GES DISC"})})
+          concept (d/item->concept granule {:format :umm-json
+                                            :version "1.4"})]
+      (vp/assert-matching-granule-urs
```

Comment:
We should verify that virtual granules are generated, not just the source granule.